File: EGIS_Management/Domain/Entities/bpltas/Bank.cs
using System.ComponentModel.DataAnnotations;

namespace Domain.Entities.bpltas
{
    public class Bank
    {
        [Key]
        public int Id { get; set; }
        public string BankName { get; set; } = string.Empty;
        public string BankCode { get; set; } = string.Empty;
        public string Branch { get; set; } = string.Empty;
        public string ContactName { get; set; } = string.Empty;
        public string ContactNo { get; set; } = string.Empty;
        public string? Address { get; set; }
        public DateTime CreatedAt { get; set; } = DateTime.Now;
    }
}

File: SPA19/EGIS/src/app/bpltas/models/bank.ts
export interface IBank {
    id: number;
    bankname: string;
    bankcode : number;
    branch : string;
    contactname : string;
    contactno : string;
    address : string;
 }

 export type createBankDto =Omit<IBank,'bankname'|'bankcode'|'branch'|'contactname'|'contactno'|'address'>;